lib/renderers/cairo_renderer.h

```diff
@@ -260,17 +262,19 @@ bm_cairo_entry_message(char *entry_text, bool highlighted, uint32_t event_feedba
 }
 
 static inline void
-bm_cairo_draw_rounded_path(cairo_t *cr, double x, double y, double width, double height, double radius)
+bm_cairo_rounded_path(cairo_t *cr, double x, double y, double width, double height, double radius)
 {
+    double ajusted_radius = MIN(MIN(radius, height*0.5), width*0.5); /* Prevent border from intersecting itself. */
```

Comment:
Is this a typo? Should it be `adjusted_radius`?

Reply:
Fixed